litmus-portal/Makefile

```diff
@@ -88,3 +88,53 @@ backend-services-checks:
 		&& exit 1; \
 	fi
 
+.PHONY: docker.buildx
+docker.buildx:
+	@echo "------------------------------"
+	@echo "--> Setting up Builder        " 
+	@echo "------------------------------"
+	@if ! docker buildx ls | grep -q multibuilder; then\
+		docker buildx create --name multibuilder;\
+		docker buildx inspect multibuilder --bootstrap;\
+		docker buildx use multibuilder;\
+		docker run --rm --privileged multiarch/qemu-user-static --reset -p yes;\
+	fi
```

Comment:
Instead of using these commands. We can use docker buildx multi builder action.
Ref- https://github.com/litmuschaos/chaos-operator/blob/bab1c1c4b08207155285cd29e3a2ecd76f3f4a34/.github/workflows/push.yml#L46